### PR TITLE
added validating status

### PR DESCRIPTION
**Pull request by chrisdugne**
### Result and observation

now displaying proper slide data during the go to correction animation, and not the next question anymore

<p align="center">
  
</p>

- [ ] **Breaking changes ?**  

- [x] **Extra dependency ?**

~~requires https://github.com/CoorpAcademy/components/pull/1482~~

### Testing Strategy

- [ ] Already covered by tests
- [x] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by chrisdugne** [comment]
[store: branchement fonctionnel: Slide + correction + vies](https://trello.com/c/TyjU3Ztz/1063-store-branchement-fonctionnel-slide-correction-vies)

**Comment by chrisdugne** [comment]
[store: branchement fonctionnel: Slide + correction + vies](https://trello.com/c/TyjU3Ztz/1063-store-branchement-fonctionnel-slide-correction-vies)

File: src/screens/correction.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ export type Params = {|
 |};
 
 type ConnectedDispatchProps = {|
-  selectProgression: () => (dispatch: Dispatch, getState: GetState) => void
+  selectProgression: () => (dispatch: Dispatch, getState: GetState) => void,
+  selectRoute: typeof selectRoute
```
Comment (VincentCATILLON) [inline src/screens/correction.js]: 👍

**Comment by VincentCATILLON** [comment]
Check lint please

**Pull request merged**